yarn.lock

```diff
@@ -6900,6 +6904,13 @@ react-banner@^1.0.0-rc.0:
   version "1.0.0-rc.0"
   resolved "https://registry.yarnpkg.com/react-banner/-/react-banner-1.0.0-rc.0.tgz#59627abaf240cdea33b81a9438d499af07aeb103"
 
+react-document-title@^2.0.3:
```

Comment:
it seems like a 1st level dependency, where did it come from?

Reply:
For some reason after the rebase I'm seeing stuff I didn't modify like this one. I will try to clean this up.